src/components/Form/Location/Location.jsx

```diff
@@ -866,6 +867,8 @@ Location.defaultProps = {
   required: false,
   addressBooks: {},
   addressBook: '',
+  disallowPoBox: false,
```

Comment:
this prop looks like it's not being used

Reply:
rats! i forgot remove that. good catch